clientUtils/Util.ts

```diff
@@ -379,26 +379,6 @@ export const domainExtent = (
     }
 }
 
-// Compound annual growth rate
-// cagr = ((new_value - old_value) ** (1 / Δt)) - 1
-// see https://en.wikipedia.org/wiki/Compound_annual_growth_rate
```

Comment:
This comment I would leave in - it's still relevant for the `cagr` method below, and it is hard to know what `cagr` stands for without this comment (which is maybe a problem in and of itself).